charts/camunda-platform/values.yaml

```diff
@@ -780,7 +780,7 @@ zeebe:
     ## @param zeebe.livenessProbe.scheme defines the startup probe schema used on calling the probePath
     scheme: HTTP
     ## @param zeebe.livenessProbe.probePath defines the liveness probe route used on the app
-    probePath: /actuator/health/liveness
+    probePath: /actuator/health/readiness
```

Comment:
:wrench: I would add a comment explaining this is not a mistake, and we indeed use readiness here explicitly. You can link to the issue for context.